packages/backend/src/database/migrations/20250821071500_add_title_to_ai_thread.ts

```diff
@@ -0,0 +1,30 @@
+import { Knex } from 'knex';
+
+const AiThreadTableName = 'ai_thread';
+
+export async function up(knex: Knex): Promise<void> {
+    const hasTitle = await knex.schema.hasColumn(AiThreadTableName, 'title');
+
+    if (hasTitle) {
+        return;
+    }
+
+    await knex.schema.alterTable(AiThreadTableName, (table) => {
+        table.string('title', 255).nullable();
```

Comment:
https://knexjs.org/guide/schema-builder.html#string
 > Adds a string column, with optional length defaulting to 255.